pkg/gatherers/workloads/workloads_gatherer.go

```diff
@@ -36,6 +38,11 @@ func (g *Gatherer) GetGatheringFunctions(context.Context) (map[string]gatherers.
 				return g.GatherWorkloadInfo(ctx)
 			},
 		},
+		"helm_info": {
+			Run: func(ctx context.Context) ([]record.Record, []error) {
+				return g.GatherHelmInfo(ctx)
+			},
+		},
```

Comment:
workloads gatherer runs only each 12h. Is it desired for this data?

Reply:
It is not, it needs to be executed once a day. I'm moving it to a separated gather.

Reply:
hmm the question is if it's worth creating a new gatherer....maybe we should check with processing team if there's some extra logic for those data which is not in every archiuve

Reply:
We will keep using the `workloads`